TESTING.md

```diff
@@ -14,6 +14,7 @@ Repository Permissions:
 - Commit statuses: read and write
 - Contents: read and write
 - Environments: read and write
+- Issues: read and write
```

Comment:
On line 37/38 below the "Issue comment" event is not available unless the Issues permission is configured first. I'm not sure if we need only read, or both read and write though. Please let me know which one is required and I can update here.

Reply:
hmm that was probably missed, but it only needs to be read only, writing is not mandatory.

Reply:
actually we should have a way to generate a GitHub app with all necessary permissions via a manifest: https://docs.github.com/en/apps/sharing-github-apps/registering-a-github-app-from-a-manifest

to simplify the process.

Reply:
I updated issues to "read only"

Reply:
> actually we should have a way to generate a GitHub app with all necessary permissions via a manifest: https://docs.github.com/en/apps/sharing-github-apps/registering-a-github-app-from-a-manifest
> 
> to simplify the process.

That sounds like a good idea. We should track that in a new issue.

Reply:
Created https://github.com/eclipse-csi/otterdog/issues/433